lib/open-amp/resource_table.h

```diff
@@ -17,8 +17,14 @@ extern "C" {
 #if (CONFIG_OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF > 0)
 
 #define VDEV_ID                 0xFF
-#define VRING0_ID 0 /* (master to remote) fixed to 0 for Linux compatibility */
-#define VRING1_ID 1 /* (remote to master) fixed to 1 for Linux compatibility */
+#define VRING0_ID                                                                                  \
+	CONFIG_OPENAMP_RSC_TABLE_IPM_RX_ID /* (master to remote) default to 0 for                  \
+					    * Linux compatibility                                  \
+					    */
+#define VRING1_ID                                                                                  \
+	CONFIG_OPENAMP_RSC_TABLE_IPM_TX_ID /* (remote to master) default to 1 for                  \
+					    * Linux compatibility                                  \
+					    */
```

Comment:
The multiline comment format is not correct and the term `master` should be removed.
Furthermore the sentence `default to 0/1 for Linux compatibility` seems deprecated now , you can remove it.

proposal: 
#define VRING0_ID  CONFIG_OPENAMP_RSC_TABLE_IPM_RX_ID /* (host to remote) */
#define VRING1_ID  CONFIG_OPENAMP_RSC_TABLE_IPM_TX_ID /* (remote to host) */